pkg/route/registry/route/rest.go

```diff
@@ -103,6 +110,13 @@ func (rs *REST) Update(ctx kapi.Context, obj runtime.Object) (<-chan apiserver.R
 	if errs := validation.ValidateRoute(route); len(errs) > 0 {
 		return nil, errors.NewInvalid("route", route.Name, errs)
 	}
+
+	err := rs.loadCerts(route.TLS)
```

Comment:
For now just force people to set the data, and have examples.  The data should be ascii safe - if it's not, that's something we need to consider.

> On Jan 14, 2015, at 10:37 PM, Paul <notifications@github.com> wrote:
> 
> In pkg/route/registry/route/rest.go:
> 
> > @@ -103,6 +110,13 @@ func (rs *REST) Update(ctx kapi.Context, obj runtime.Object) (<-chan apiserver.R
> >  	if errs := validation.ValidateRoute(route); len(errs) > 0 {
> >  		return nil, errors.NewInvalid("route", route.Name, errs)
> >  	}
> > +
> > +	err := rs.loadCerts(route.TLS)
> I've called this out as needing an update and asked @fabianofranz how it is best implemented in the cli. #652 (comment) This is certainly not intended to go in as-is since the cert will not be located on the server side
> 
> —
> Reply to this email directly or view it on GitHub.
>